export const signUp = async (req , res , next)=>{
    try {

    } catch (error) {
        
    }
}

export const signIn = async (req , res , next)=>{
    try {
        
    } catch (error) {
        
    }
}
export const signOut = async (req , res , next)=>{
    try {
        
    } catch (error) {
        
    }
}
export const me = async (req , res , next)=>{
    try {
        
    } catch (error) {
        
    }
}
export const forgotPassword = async (req , res , next)=>{
    try {
        
    } catch (error) {
        
    }
}
export const resetPassword = async (req , res , next)=>{
    try {
        
    } catch (error) {
        
    }
}
export const changePassword = async (req , res , next)=>{
    try {
        
    } catch (error) {
        
    }
}
export const verifyCode = async (req , res , next)=>{
    try {
        
    } catch (error) {
        
    }
}